android/src/com/mapswithme/maps/routing/SlotFrame.java

```diff
@@ -99,7 +100,7 @@ private void updateText()
       if (MapObject.isOfType(MapObject.MY_POSITION, mMapObject))
         mText.setText(R.string.p2p_your_location);
       else
-        mText.setText(mMapObject.getTitle());
+        mText.setText(!TextUtils.isEmpty(mMapObject.getTitle()) ? mMapObject.getTitle() : mMapObject.getSubtitle());
```

Comment:
Пустой для не-фич, например, для координат или лонг тапа на пустом месте

> On 25.3.2016, at 21.33, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/routing/SlotFrame.java:
> 
> > @@ -99,7 +100,7 @@ private void updateText()
> >        if (MapObject.isOfType(MapObject.MY_POSITION, mMapObject))
> >          mText.setText(R.string.p2p_your_location);
> >        else
> > -        mText.setText(mMapObject.getTitle());
> > +        mText.setText(!TextUtils.isEmpty(mMapObject.getTitle()) ? mMapObject.getTitle() : mMapObject.getSubtitle());
> Лучше обратить условие. А getSubtitle() всегда непустой?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>